src/app/CommandSender.h

```diff
@@ -375,6 +376,26 @@ class CommandSender final : public Messaging::ExchangeDelegate
 
     TLV::TLVWriter * GetCommandDataIBTLVWriter();
 
+    /**
+     * API for adding request data.  The `aEncodable` is generally expected to encode
+     * a ClusterName::Commands::CommandName::Type struct, however any object should work.
```

Comment:
This doesn't really make sense.  If you have a struct, you'd just call the templated API...